app/views/content/is-teaching-right-for-me/_why-teach.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="why-teach" aria-label="Why teach?">
+  <div class="why-teach__container">
+    <div class="why-teach__content">
+      <%= render Content::HeadingComponent.new(
+        heading: "A job like no other",
+        tag: :h2,
+        caption: "Looking for challenge and variety?",
+        caption_size: :l
+      ) %>
+
+      <p>
+        As a teacher you'll work with lots of young people with a range of abilities, thinking on your feet to find creative solutions to problems.
+      </p>
+
+      <ul>
+        <li><%= link_to("Read how Zainab works alongside colleagues to support her pupils", "/blog/school-experience-helped-me-decide-to-switch") %></li>
+        <li><%= link_to("Read how Gavin adapts his teaching to create a positive learning environment.", "/blog/no-two-days-are-the-same") %></li>
```

Comment:
full stop needs moving outside of the link (and there's also not a full stop in the above link text but is one here)